src/components/InputItem/InputItem.module.css

```diff
@@ -0,0 +1,7 @@
+.inputWrap {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.inputField { width: 80%; }
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/tr-todo/pullRequest?prid=5765946)